server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -51,19 +45,17 @@ def test_process_tb_file_not_found_error(self, monkeypatch, caplog, test_logger)
         tb_path = fixture_path / "bad_tarball"
 
         def mock_get(self, section: str, option: str):
-            if option == "pbench-receive-dir-prefix":
-                return tb_path
-            if option == "put-token":
-                return "Authorization-token"
-            assert (
-                False
-            ), f"Unexpected configuration option, {section!r}/{option!r}, in mocked config.get()"
+            assert option in ["pbench-receive-dir-prefix", "put-token"]
+            return (
+                tb_path
+                if option == "pbench-receive-dir-prefix"
+                else "Authorization-token"
+            )
```

Comment:
OK, well, `black` makes this a bit less neat than I might have hoped. 😬 But I still like it better than the multiple `if` statements, so we're ahead.

Reply:
yeah, I like it better now :D

Reply:
Yeah, Black is slowly weaning me away from my love for ternary expressions (although, part of that is the fact that they are ugly in Python to begin with, but Black's insistence that the pieces all go on separate lines with equal indentation takes them from ugly to hard-to-read...).